modules/apps/static/portal-configuration/portal-configuration-persistence-test/src/testIntegration/java/com/liferay/portal/configuration/persistence/listener/test/ConfigurationModelListenerTest.java

```diff
@@ -222,13 +222,13 @@ public void onBeforeSave(
 		}
 	}
 
-	private static Configuration _getConfiguration(String pid, String location)
+	private static Configuration _getConfiguration(String pid)
 		throws IOException {
 
 		return OSGiServiceUtil.callService(
 			_bundleContext, ConfigurationAdmin.class,
 			configurationAdmin -> configurationAdmin.getConfiguration(
-				pid, location));
+				pid, StringPool.QUESTION));
```

Comment:
When we create a method, no matter it is public or not, we want to make sure the parameters are really used for something. In this case, the "location" is not really used, as we always pass in "?". Then we should not have this parameter, just use "?" directly.